Extensions/retags.js

```diff
@@ -20,16 +20,24 @@ XKit.extensions.retags = {
 		} catch(e) {}
 		this.add_toggle();
 		this.observer.observe($('body')[0],{childList:true,subtree:true});
-		this.tag(this.selectors);
+		XKit.extensions.retags.tag(XKit.extensions.retags.selectors);
```

Comment:
Ah. No reason, I was fiddling around with other changes here and removed this line and then re-added it by hand without thinking about it. I'll put it back  to how it was for consistency's sake.